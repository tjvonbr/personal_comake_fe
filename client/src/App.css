.App {
  text-align: center;
}


.register-button {
  margin-left: 10px!important;
}

.nav-container {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
}