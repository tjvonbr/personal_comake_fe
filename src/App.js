import React from 'react';
import { Route } from 'react-router-dom';
import { ThemeProvider } from 'emotion-theming';
import Register from './components/Register';
import Login from './components/Login';
import Profile from './components/Profile';
import ProfileEdit from './components/ProfileEdit';
import AddIssue from './components/AddIssue';
import Community from './components/Community';
import Landing from './components/Landing';
import Logout from './components/Logout';
import NewRegister from './components/NewRegister';

// Theme colors
const theme = {
  colors: {
    white: '#FBFAFB',
    lightBlue: '#5D91BS',
    green: '#8BDDA1',
    turquoise: '#5D9B87',
    blue: '#21316C'
  }
}

function App() {
  return (
    <ThemeProvider theme={theme}>
      <div className="App">
        <Route exact path="/" component={Landing} />
        <Route exact path="/profile/:id" component={Profile} />
        <Route path="/profile/:id/edit" component={ProfileEdit} />
        <Route path="/login" component={Login} />
        <Route path="/register" component={NewRegister} />
        <Route path="/addissue" component={AddIssue} />
        <Route path="/community/:zipcode" component={Community} />
        <Route path="/logout" component={Logout} />
      </div>
    </ThemeProvider>
  );
}

export default App;
