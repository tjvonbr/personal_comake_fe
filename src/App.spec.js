import React from 'react';
import renderer from 'react-test-renderer'

import App from './App';

describe('My Test Suite', () => {
  it('My Test Case', () => {
    expect(true).toEqual(true);
  });
});

describe('App', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});