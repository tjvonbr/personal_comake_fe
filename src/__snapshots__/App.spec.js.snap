// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<div
  className="App"
>
  <div
    style={
      Object {
        "background": "#f6f9fc",
      }
    }
  >
    <header
      className="landing-cta"
    >
      <nav
        className="nav-wrapper"
      >
        <h1
          className="nav-vase-logo white"
        >
          Vase
        </h1>
        <a
          aria-current={null}
          className="nav-link"
          href="/login"
          onClick={[Function]}
        >
          Sign In
        </a>
      </nav>
      <div
        className="cta-content-wrapper"
      >
        <h1>
          A new way to engage
        </h1>
        <p
          className="landing-cta-content"
        >
          Vase is the smart approach to engaging with your neighbors. See what others are up to on your block. Check in to see what issues within your community need to be addressed. Give your local officials an idea of what concerns you most. Vase turns the web into a virtual town hall. How will you help build your community?
        </p>
        <button
          className="btn-cta"
          onClick={[Function]}
        >
          Get Started
        </button>
      </div>
      <div
        id="device-container"
      >
        <div
          className="css-mx9hnp-PhoneIcon"
        >
          <div
            className="css-407xgl-PhoneIcon"
          >
            <div
              className="css-44wwz4-PhoneIcon"
            >
              <embed
                alt="iphone icon"
                src="iphone.svg"
              />
            </div>
          </div>
        </div>
        <div
          className="css-iu2rbe-TabletIcon"
        >
          <div
            className="css-13hunaa-TabletIcon"
          >
            <div
              className="css-nvxig3-TabletIcon"
            >
              <embed
                alt="tablet icon"
                src="tablet.svg"
              />
            </div>
          </div>
        </div>
      </div>
    </header>
    <section
      className="landing-features"
    >
      <div
        className="landing-features-outer-wrapper"
      >
        <h1>
          Build your community
        </h1>
        <div
          className="landing-features-inner-wrapper"
        >
          <section
            className="landing-features-template"
          >
            <h2>
              Raise local awareness
            </h2>
            <p>
              We believe in democratic participation at all levels of government, and we understand that not everyone has time to attend town hallmeetings, public demonstrations, or even vote for local offices.  Make yourvoice heard by creating community concerns and allow others to upvote it to let local officialsknow what you deem important.
            </p>
            <form
              className="form-landing"
              onSubmit={[Function]}
            >
              <label>
                <input
                  className="input-landing"
                  name="zipcode"
                  onChange={[Function]}
                  placeholder="Enter your city's zipcode"
                  type="text"
                  value=""
                />
              </label>
              <button
                id="btn-zip"
                onClick={[Function]}
              >
                Go!
              </button>
            </form>
          </section>
          <section
            className="landing-features-template"
          >
            <h2>
              Organize community events
            </h2>
            <p>
              Want to create a community event in your city? March for something you believe in.  Demonstrate for what you believe is right.  If your message needs to be heard, we'll provide the platform.
            </p>
          </section>
        </div>
      </div>
    </section>
  </div>
</div>
`;
